app/src/main/java/org/android/go/sopt/ListUsersService.kt

```diff
@@ -0,0 +1,11 @@
+package org.android.go.sopt
+
+
+import org.android.go.sopt.home.ResponseListUsersDto
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface ListUsersService {
+    @GET("/api/users?.page=2")
```

Comment:
```suggestion
@GET("/api/users")
   fun listUsers(
        @Query("page") num: Int = 2
    ): Call<ResponseListUsersDto>
```
좀 더 가독성 있게 API 요청을 하기 위해서는 `@Query`를 이용해서 작성하는 방법도 있답니다 !

Reply:
오? 이거 정말 좋네요!!